<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>399</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>423</width>
    <height>399</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>423</width>
    <height>399</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BlackJack</string>
  </property>
  <widget class="QPushButton" name="play_but">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Играть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="achivbut">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>140</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Достижения</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_but">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
